avm/res/cache/redis/tests/e2e/entra-id/main.test.bicep

```diff
@@ -31,6 +31,15 @@ resource resourceGroup 'Microsoft.Resources/resourceGroups@2021-04-01' = {
   location: resourceLocation
 }
 
+module dependencies 'dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-dependencies'
```

Comment:
Update to

`name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'`